Introduce specific event replay command

arothuis opened the pull request:
[133173121](https://www.pivotaltracker.com/story/show/133173121)

This new command is based on the ReplayEventsCommand that runs a replayer for all events, which was primarily used for development purposes.
Its purpose is to replay specific events for specific projectors.

arothuis commented on src/Surfnet/StepupMiddleware/MiddlewareBundle/Console/Command/ReplaySpecificEventsCommand.php:
```diff
@@ -0,0 +1,72 @@
+    protected function configure()
+    {
+        $this
+            ->setName('middleware:event:specific_replay')
```
Not sure about the name. I would prefer `middleware:event:replay`, but that one is already taken for the deprecated/development replay command. I could rename or remove that one, but that feels odd when there is no replacement in place.

Thoughts?

DRvanR replied:
`stepup:events:replay` (domain:component-of-action:action) - kinda aligns with the `stepup:migrate:institution-configuration`

I feel using the `stepup:` as new base NS is a good differentiator of intent with the existing `middleware` ones.

arothuis replied:
Good idea.

DRvanR commented on src/Surfnet/StepupMiddleware/MiddlewareBundle/Console/Command/ReplaySpecificEventsCommand.php:
```diff
@@ -0,0 +1,72 @@
+            '',
+            'WARNING!!!!',
+            '',
+            'You are about to WIPE all read data and recreate all data based on chosen events and projectors.',
```
This should plain simply not be true. As discussed yesterday - this is intended to be a proper event replay, through specific replay-projectors.

In general: please remove the copy pasted code. For now just a message that it does not actually do anything should suffice :wink:

arothuis replied:
OK.

arothuis commented:
I have removed all `execute()` code for now and moved it to `stepup:event:replay`. 

Do note that the command name is still `ReplaySpecificEvents` as it would conflict with `ReplayEvents`. IMHO, this is fine.
